sigtrap_posix.go

```diff
@@ -25,58 +25,53 @@ import (
 
 // trapSignalsPosix captures POSIX-only signals.
 func trapSignalsPosix() {
+	signalToString := map[os.Signal]string{syscall.SIGHUP: "SIGHUP", syscall.SIGUSR1: "SIGUSR1"}
+
 	go func() {
 		sigchan := make(chan os.Signal, 1)
 		signal.Notify(sigchan, syscall.SIGTERM, syscall.SIGHUP, syscall.SIGQUIT, syscall.SIGUSR1, syscall.SIGUSR2)
 
 		for sig := range sigchan {
 			switch sig {
-			case syscall.SIGTERM:
-				log.Println("[INFO] SIGTERM: Terminating process")
+			case syscall.SIGQUIT:
+				log.Println("[INFO] SIGQUIT: Terminating process")
 				if PidFile != "" {
 					os.Remove(PidFile)
 				}
 				os.Exit(0)
 
-			case syscall.SIGQUIT:
-				log.Println("[INFO] SIGQUIT: Shutting down")
-				exitCode := executeShutdownCallbacks("SIGQUIT")
+			case syscall.SIGTERM:
+				log.Println("[INFO] SIGTERM: Shutting down")
+				exitCode := executeShutdownCallbacks(signalToString[sig])
```

Comment:
signalToString map doesn't have syscall.SIGTERM key

Reply:
Good point, I'll fix this locally and push soon. Thanks!